views/partials/footer.tmpl

```diff
@@ -1,2 +1,8 @@
 {{define "footer"}}
+	<footer class="container outer footer">
+        	<div class="navbar-left"><a href="#">Disclaimer</a> | <a href="#">MyRadio</a> | <a href="#">Contact</a></div>
+        	<div class="navbar-right">
+				<p>Copyright © 1967-(Insert Current Year Here) University Radio York. Designed by the <a href="mailto:webmaster@ury.org.uk" title="Click to email the Computing team.">URY Computing Team</a></p>
```

Comment:
We will have to "hardcode" in the current year here anyway. Fairly sure automatically updating copyright notices don't work :p